app/assets/src/components/views/discovery/discovery_sidebar.scss

```diff
@@ -32,11 +33,18 @@
 
     color: $black;
     font-weight: 600;
-    letter-spacing: 1.45px;
+    font-size: 12px;
```

Comment:
There should be a typography mixin for this size no? 
If there is something specific to this one, can you create a new one?